cpp/include/ray/core.h

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
what's this file for, should this be combined into another file if it only contains a header?